rustsec/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name        = "rustsec"
 description = "Client library for the RustSec security advisory database"
-version     = "0.23.3" # Also update html_root_url in lib.rs when bumping this
+version     = "0.24.0" # Also update html_root_url in lib.rs when bumping this
```

Comment:
Usually I reserve these sort of bumps for a release commit... how about:

```suggestion
version     = "0.24.0-pre" # Also update html_root_url in lib.rs when bumping this
```

Reply:
Renamed to `0.24.0-beta.1`